Gruntfile.js

```diff
@@ -97,11 +97,17 @@ module.exports = function(grunt) {
       tasks: 'dev'
     },
     connect: {
-      dev: {
+      preview: {
```

Comment:
Yes, I just did it this way to preserve the old way so I wouldn't surprise people. We can definitely just combine everything.